cli/build/build.go

```diff
@@ -60,19 +91,31 @@ func New() *cobra.Command {
 				log.WithError(err).Fatal("Failed to load compose file")
 			}
 
+			builder, err := getImageBuilder(regCreds, dockerConfig, authConfig.AuthToken, forceBuildkit)
+			if err != nil {
+				log.WithError(err).Fatal("Get image builder")
+			}
+
+			buildOpts := map[string]build.BuildPushConfig{}
 			for _, svc := range parsedCompose.Services {
 				if svc.Build == nil {
 					log.Infof("%s uses an image, skipping\n", svc.Name)
 					continue
 				}
 
-				log.Infof("Building %s\n", svc.Name)
```

Comment:
Will these logs still show up? (for both buildkit and docker)

Reply:
Buildkit shows it as part of their UI. I made sure to add it to the docker builder.